project-codeflare/codeflare-sdk
# Upgrade Ray to version 2.35

# Issue link


[RHOAIENG-12186](https://issues.redhat.com/browse/RHOAIENG-12186) 

# What changes have been made

All instances of the ray version and image have been updated to Ray 2.35 and `quay.io/modh/ray:2.35.0-py39-cu121`

# Verification steps



- Install RHOAI 2.14 nightly build (brew.registry.redhat.io/rh-osbs/iib:812075)
- Run the interactive demo notebooks from this PR and enure that they work as expected.

## Checks
- Testing Strategy
